### PR TITLE
LPS-172472 Set the window mode to undefined

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutActionsDisplayContext.java [hunk 1]
```diff
@@ -273,7 +275,11 @@ private String _getPreviewLayoutURL(Layout layout) {
 			"segmentsExperienceId",
 			String.valueOf(_getSegmentsExperienceId(draftLayout)));
 
-		return getPreviewLayoutURL.toString();
+		String url = HttpComponentsUtil.removeParameter(
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutActionsDisplayContext.java]: Nice catch! I've updated it. Many thanks @jkappler

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/128638

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**